Contributing.md

```diff
@@ -0,0 +1,96 @@
+add appropriate metadata before the issue is addressed.
+
+Please be courteous, respectful, and every participant is expected to follow the
+project's Code of Conduct.
+
+# Contributions
+
+Any change to resources in this repository must be through pull requests. This applies to all changes
+to documentation, code, binary files, etc. Even long term collaborators and WG members must use
+pull requests.
+
+No pull request can be merged without being reviewed.
+
+For non-trivial contributions, pull requests should sit for at least 36 hours to ensure that
+contributors in other timezones have time to review. Consideration should also be given to
+weekends and other holiday periods to ensure active collaborators all have reasonable time to
+become involved in the discussion and review process if they wish.
+
+The default for each contribution is that it is accepted once no collaborator has an objection.
+During review collaborators may also request that a specific contributor who is most versed in a
+particular area gives a "LGTM" before the PR can be merged. There is no additional "sign off"
+process for contributions to land. Once all issues brought by collaborators are addressed it can
+be landed by any collaborator.
+
+In the case of an objection being raised in a pull request by another collaborator, all involved
+collaborators should seek to arrive at a consensus by way of addressing concerns being expressed
+by discussion, compromise on the proposed change, or withdrawal of the proposed change.
+
+If a contribution is controversial and collaborators cannot agree about how to get it to land
+or if it should land then it should be escalated to the WG. WG members should regularly
+discuss pending contributions in order to find a resolution. It is expected that only a
+small minority of issues be brought to the WG for resolution and that discussion and
+compromise among collaborators be the default resolution mechanism.
+
+# Becoming a Member and Collaborator
+
+To become a member of the Community Committee, start participating! Attend the bi-weekly meetings, investigate issues tagged as `good first issue`,
+file issues and pull requests, and provide insight via GitHub. Request to become an Observer by
+filing an issue. Once added as an Observer to meetings, we will track attendance and participation
+for 3 months, as according to our [governance guidelines](https://github.com/nodejs/community-committee/blob/master/GOVERNANCE.md#section-4-establishment-of-the-community-committee). If you meet the minimum
+attendance and are participating, the CommComm will vote to add you as a member.
+
+All participants added as members should be on-boarded in a timely manner, added as a collaborator, and be given write access to the repository.
+
+Collaborators are expected to follow this policy and continue to send pull requests, go through
+proper review, and have other collaborators merge their pull requests.
+
+# The CommComm Process
+
```

Comment:
believe this should be case in the context of the user-feedback committee.  Ie The user-feedback committee follows the process of the Community Committee which uses "consensus seeking" process ....

Reply:
Thank you for the comments/review. I'll take it and work on  patch-2 branch on mihaiep/user-feedback.